applications/nrf5340_audio/src/utils/fw_info_app.c.in

```diff
@@ -20,9 +20,9 @@ static const char COMPILE_DATE[]    = "@NRF5340_AUDIO_CORE_APP_COMP_DATE@";
 static const char NRF5340_CORE[]    = "nRF5340 Audio nRF5340 Audio DK cpuapp";
```

Comment:
fw_info_app.c.in does not look clang-formatted. This is probably because .in. 
Maybe remove .in, do clang-format and then add .in again?

Reply:
Done